lib/common/output_log.c

```diff
@@ -248,6 +248,29 @@ log_info(pcmk__output_t *out, const char *format, ...) {
     return pcmk_rc_ok;
 }
 
+G_GNUC_PRINTF(2, 3)
+static int
+log_transient(pcmk__output_t *out, const char *format, ...)
+{
+    private_data_t *priv = NULL;
+    int len = 0;
+    va_list ap;
+    char* buffer = NULL;
```

Comment:
This was all copied directly from `log_info()`, the only change being the log priority. I'll move the asterisk after review.